cfg/cis-1.5/master.yaml

```diff
@@ -297,8 +304,15 @@ groups:
 
       - id: 1.1.20
         text: "Ensure that the Kubernetes PKI certificate file permissions are set to 644 or more restrictive (Scored)"
-        audit: "stat -c %n\ %a /etc/kubernetes/pki/*.crt"
-        type: "manual"
+        audit: "stat -c permissions=%a /etc/kubernetes/pki/*.crt"
```

Comment:
Hey, I tested it and notice it will not check sub directories, better use find, like the first test you changed 1.1.19 
It also the audit commend in the new CIS version 
![image](https://user-images.githubusercontent.com/12915002/89404269-57325e00-d722-11ea-850e-1a44bbc2aa21.png)

Reply:
Looks like the new CIS version which you mean is cis 1.5.1, but this PR is talk about cis 1.5, maybe we should follow the cis 1.5 instead of cis 1.5.1?